configs/server/forseti_conf_server.yaml.in

```diff
@@ -215,6 +209,15 @@ scanner:
 
 notifier:
 
+    # Provide connector details to receive violations when found
+    email_connector_config:
```

Comment:
Call this more succinctly: `email_connectors`.  The config is repetitive.

Reply:
Changed it to `email_connector` in config file and `email_connector_config` in the code as discussed.